.github/workflows/tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-          python-version: [3.6, 3.7, 3.8]
+          python-version: [3.7, 3.8, 3.9]
```

Comment:
Minor: for the line below this one, in general we are removing the tests of `min`, keeping only the test for the latest. See for example [here](https://github.com/inveniosoftware/invenio-communities/blob/master/.github/workflows/tests.yml#L33).

Reply:
Awesome-that will make maintenance easier!